### PR TITLE
Publish to available name.

### PR DESCRIPTION
**Pull request by theengineear**
The prior namespace was no longer under our control. Additionally, this new name is less redundant since “@netflix” is already the scope.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
   },
   "homepage": "https://github.com/Netflix/eslint-config-netflix",
   "peerDependencies": {
-    "eslint": "^8.57.0"
+    "@eslint/js": "^8.x"
```
Comment (theengineear) [inline package.json:33]: This is really all we depend on. We want this plugin to attempt to be as flexible as possible to prevent version conflicts.

**Comment by theengineear** [comment]
Still just working towards #22.

File: package.json (hunk 1 above)
Comment (klebba) [reply package.json:33]: I think we would require `8.21` of ESLint which is when the flat config format was introduced (still trying to find some reference about that...)

Comment (klebba) [reply package.json:33]: * also ESLint/js may not have the same version alignment ...

Comment (theengineear) [reply package.json:33]: I can just go back to `^8.57.0` as well.

Comment (theengineear) [reply package.json]: > also ESLint/js may not have the same version alignment ...

Comment (klebba) [reply package.json]: here's the release notes announcing `8.21` is the version we care about: https://eslint.org/blog/2022/08/eslint-v8.21.0-released/

**Comment by klebba** [comment]
We need to update the package name here: https://github.com/Netflix/eslint-config-netflix/blob/main/README.md — let me see if there's anywhere else

**Pull request merged**